exercises/concept/guessing-game/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+# Hints
+
+## General
+
+- [This page][pattern-matching] has a nice introduction to pattern matching in Haskell.
```

Comment:
```suggestion
- Learn You A Haskell has a nice [introduction to pattern matching in Haskell][pattern-matching].
```

We aren't allowed to use "this page" because of https://www.w3.org/QA/Tips/noClickHere. Here is one suggestion for how we can follow the rules. However, other ideas would work as well.